src/software/gui/robot_diagnostics/widgets/robot_diagnostics_gui.cpp

```diff
@@ -13,7 +13,7 @@ RobotDiagnosticsGUI::RobotDiagnosticsGUI(
       update_timer(new QTimer(this)),
       push_primitive_timer(new QTimer(this)),
       update_timer_interval(Duration::fromSeconds(1.0 / 60.0)),
-      push_primitive_timer_interval(Duration::fromSeconds(1.0 / 60.0))
+      push_primitive_timer_interval(Duration::fromSeconds(1.0 / 30.0))
```

Comment:
just wondering why 1/30

Reply:
because the interval in the wrapper is 1/60 so the buffer would get full and stay full